src/init-nmos.js

```diff
@@ -48,9 +49,13 @@ function getPrioritised (representations, priority, version, protocol) {
   if (priority) {
     let representation = representations
       .filter(representation => {
+        if (!representation.txt.hasOwnProperty('api_auth')) {
+          representation.txt.api_auth = false
```

Comment:
The new mDNS bridge has an 'authorization' parameter which should avoid the need to index into the txt records.